### ISSUE TITLE
Party mode does not work properly

### ISSUE THREAD

**Issue by lelemaus** [UltraStar-Deluxe/USDX#443]
**Please, do not create duplicate issues**

### Actual behaviour

When starting party mode with default settigs, the only song that comes up is the first one in my library. Even when using a joker only this song is singalbe.
When choosing a playlist in party mode settings the game freezes or crashes as soon as the first round would start (see crash report attached)

### Expected behaviour

All songs available / no crash

### Steps to reproduce

Not sure if possible, since I could not find anyone complaining about the same problem...

### Details

Provide some additional information:

- USDX version: V 2017.8.0 dev Build
- Operating System + version: Win 10 64 bit

Crash report after trying party mode with playlist selection:
![us crash playlist deutsch](https://user-images.githubusercontent.com/47500085/52535064-0a96de00-2d4a-11e9-8dd7-20c9eb41d89a.JPG)

**Comment by lelemaus** [comment]
I've got an addition to make... I just realized the main problem is most likely that the function of random song choice (that shortcut R thing) does not work for my USDX. It always brings me to the 1st song in library and after hitting R several times because I was really annoyed the game crashed XD

![usdx crash after several random attempts](https://user-images.githubusercontent.com/47500085/52535364-c9083200-2d4d-11e9-929f-ce385f1c60d7.JPG)

**Comment by reimgun** [comment]
i have tested it a little bit and i see that this behavior with the first song is only when you have configure your song menu to Chessboard, List or Tile. With the other song menu`s the joker function works.

**Comment by dgruss** [comment]
works on current master. probably fixed since #1128

**Comment by barbeque-squared** [comment]
Yes this will be in the upcoming release 2026.3.0. Or download a current master [here](https://github.com/UltraStar-Deluxe/USDX/actions?query=branch%3Amaster+event%3Apush) and have it today.

Feel free to reopen if it persists.
